### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by JakobTopholt**
First and final commit, for some reason it loos like I changed more than I did.

F-klub is a ferrari club, F-klubben is the one we are.

### CODE REVIEW

**Comment by krestenlaust** [comment]
The formatting looks great, but I don't agree that F-club is the Ferrari club. We commonly refer to ourselves as the F-club in english. For example, the emails we send to our users in stregsystem, and our domain name too.

**Comment by JakobTopholt** [comment]
http://f-klub.dk/

**Comment by krestenlaust** [comment]
I know the domain already, you think I'm a rookie😏 doesn't change the fact that we commonly refer to F-Klubben as "the F-club" in english

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
-Sangbogen
-========
+# Sangbogen
 
-This is the current songbook in the F-Klub
+This is the current songbook of F-klubben.
```
Comment (krestenlaust) [inline README.md:3]:
```suggestion
This is the current songbook in the F-Club.
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+```
+
 3. Build the sangbog
-  `make booklet` for booklet format, `make pdf` for non-booklet (continuous) format
+   `make booklet` for booklet format, `make pdf` for non-booklet (continuous) format
 
+## Adding new songs
+
+
+=======
```
Comment (krestenlaust) [inline README.md:31]: billede
I'm not sure how this is supposed to look

File: README.md (hunk 1 above)
Comment (krestenlaust) [reply README.md:3]: F-club*

Comment (krestenlaust) [reply README.md:3]: I give up

File: README.md (hunk 2 above)
Comment (JakobTopholt) [reply README.md:31]: Ooops, Imma fix it.

**Comment by krestenlaust** [comment]
https://github.com/f-klubben/sangbog/tree/30-fix-readme

**Pull request closed without merging**